controllers/volunteer.controller.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+const Services = {
+    Volunteer: require("../services/volunteer.service"),
+    Logger: require("../services/logger.service")
+};
+const Util = require("../middlewares/util.middleware");
+
+/**
+ * @async
+ * @function createVolunteer
+ * @param {body: {volunteerDetails: {_id: MongoID, accountId: MongoID}}} req
+ * @param {*} res
+ * @return {JSON} Success or error status
+ * @description create a volunteer from information in req.body.volunteerDetails
+ */
+async function createVolunteer(req, res) {
+    const volunteerDetails = req.body.volunteerDetails;
+
+    const success = await Services.Volunteer.createVolunteer(volunteerDetails);
+
+    if (success) {
+        return res.status(200).json({
+            message: "Volunteer creation successful",
+            data: volunteerDetails
+        });
+    } else {
+        return res.status(400).json({
+            message: "Issue with volunteer creation",
+            data: {}
+        });
+    }
+}
+
+module.exports = {
+    defaultReturn: function (req, res) {
```

Comment:
What's the purpose of `defaultReturn`? I see this in a lot of your controllers.

Reply:
I'm really not sure. I remember suddenly seeing it in the exports of one of my controllers, and I thought there was a specific reason, so I started putting it in all the exports. I'll get rid of them.